### DISCUSSION TITLE
Flow path graph for C#

### DISCUSSION THREAD

**Discussion by tardigrade-9** [github/codeql#16707] [Q&A]
How do I import Flow::PathGraph for C# custom queries, I'm getting module not found. Should I install the pack?

```
---
library: false
warnOnImplicitThis: false
name: getting-started/codeql-extra-queries-csharp
version: 1.0.0
dependencies:
  codeql/csharp-all: ^1.0.0
```
codeql-pack.yml ^

**Comment by smowton** [comment]
Could you post your QL source code, and let us know how you're using CodeQL, via the CLI or VSCode or something else?
